tests/ltp/patches/listxattr03.patch

```diff
@@ -0,0 +1,73 @@
+diff --git a/testcases/kernel/syscalls/listxattr/listxattr03.c b/testcases/kernel/syscalls/listxattr/listxattr03.c
+index 4c1e7f16c..e20a19b5d 100644
+--- a/testcases/kernel/syscalls/listxattr/listxattr03.c
++++ b/testcases/kernel/syscalls/listxattr/listxattr03.c
+@@ -12,12 +12,17 @@
+ * of extended attribute names, which can be used to estimate a suitable buffer.
+ */
+ 
++/* Currently xattr is not enabled while mounting root file system. Patch is
++ * to mount root file system with xattr enabled and then use it for the test.
++*/
++
++#include <stdio.h>
+ #include "config.h"
+ #include <errno.h>
+ #include <sys/types.h>
+ 
+ #ifdef HAVE_SYS_XATTR_H
+-# include <sys/xattr.h>
++#include <sys/xattr.h>
```

Comment:
not needed

Reply:
@letmaik I asked this to be fixed since #include looks more proper than # include. Don't you aggree? Do we really need to take it back?

Reply:
What does 'looks more proper' mean?  A lot of coding style guides ask you to indent directives and, according to the C preprocessor tokenisation rules, the only valid way of doing this is to start the line with `#` and put whitespace between the `#` and the `include` or `define` token. This `#include` is inside an `#ifdef` block, so should be indented if the project's style guides say to indent preprocessor directives.  Changing it makes the code convey less local semantic information (I now need to read two lines to know that the `#include` is inside an `#ifdef` and not one).  It is a change that has no impact on functionality and decreases readability.  It should not be made.

Reply:
updated as per maik  comments. Thanks David for feedback.

Reply:
@davidchisnall @letmaik I didn't realize space after # is used for indentation for better readability under ifdef block. May bad. You are right, my comment doesn't make sense.